crates/bevy_text/src/text2d.rs

```diff
@@ -81,16 +81,16 @@ pub fn extract_text2d_sprite(
                 let rect = atlas.textures[index];
                 let atlas_size = Some(atlas.size);
 
-                let transform =
-                    Mat4::from_rotation_translation(transform.rotation, transform.translation)
-                        * Mat4::from_scale(transform.scale / scale_factor)
-                        * Mat4::from_translation(
-                            alignment_offset * scale_factor + text_glyph.position.extend(0.),
-                        );
+                let mut text_transform = transform.clone();
```

Comment:
Does this change break text rotation? The information no longer seems to exist.

Reply:
No... it's cloning the original transform, which preserves everything.

Also, I ran the example (after the commit fixing it), and it works. ;)